person_class_obj.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, parent_permission: true, name: 'Unknown')
```

Comment:
- [ ] Great job so far implementing the `Person` Class. However, I think you might have missed out to specify that the `:name` and `:age` should be the only instances that can be accessed and modified using the `attr_accessor` , and the `:id` instance should be only accessed and not modified using the `attr_reader`.

_I hope this makes sense to you_ 😄

Reply:
@MahmoudBakr23 

Thank you for reviewing my work, be sure suggested changes will be implemented.